src/global.js

```diff
@@ -119,3 +121,20 @@ export function getDisableSetCookie(): boolean {
 
   return false;
 }
+
+export function getExperimentation(): Experimentation | null {
+  if (typeof __EXPERIMENTATION__ !== "undefined") {
```

Comment:
Should we ensure this method always returns a value when `__EXPERIMENTATION__` is falsy e.g.
```json5
{
    experience: "",
    treatment: ""
}
```
This would avoid the clients of this method to do any null checks.